tutorial/connect-from-aks/ConnectFromAKS/README.md

```diff
@@ -0,0 +1,6 @@
+# Connect to Azure Cache for Redis from your AKS hosted application
+
+## Overview
+
+This repo contains code samples for connecting to Azure Cache for Redis from your AKS hosted application using user assigned managed identity or access key.
+See the [accompanying article](https://docs.microsoft.com/azure/azure-cache-for-redis/cache-tutorial-aks-get-started) for detailed instructions on how to configure your AKS cluster and run this sample.
```

Comment:
The linked doc talks about a different sample project, and doesn't describe how to configure Entra ID auth (yet)

Reply:
Yes, the doc changes are also in PR in parallel.